adhocracy4/polls/static/PollDashboard/EditPollManagement.jsx

```diff
@@ -208,10 +208,11 @@ export const EditPollManagement = (props) => {
       onSubmit={(e) => handleSubmit(e)} onChange={() => removeAlert()}
       className="editpoll__questions"
     >
-      <div>
-        <label htmlFor="allowUnregisteredUsersCheckbox">Allow unregistered user to vote</label>
-        <input type="checkbox" id="allowUnregisteredUsersCheckbox" onChange={() => setAllowUnregisteredUsers((state) => !state)} checked={allowUnregisteredUsers} />
-      </div>
+      {props.enableUnregisteredUsers &&
+        <div>
+          <label htmlFor="allowUnregisteredUsersCheckbox">Allow unregistered user to vote</label>
+          <input type="checkbox" id="allowUnregisteredUsersCheckbox" onChange={() => setAllowUnregisteredUsers((state) => !state)} checked={allowUnregisteredUsers} />
```

Comment:
we could use a couple more attributes:
aria-checked={allowUnregisteredUsers} // Adds an ARIA attribute for screen readers
aria-label="Toggle allowing unregistered users to vote"

Reply:
@hom3mad3 should we collect these things for the ui refactor story?